olive/passes/onnx/model_builder.py

```diff
@@ -197,6 +197,16 @@ def _run_for_config(
             split_assignments = model_attributes.get("split_assignments")
             if split_assignments:
                 split_assignment_str = ";".join([f"{k}={v}" for k, v in split_assignments.items()])
+                replacements = {
+                    "self_attn": "attn",
+                    "self_attention": "attn",
+                    "dense_4h_to_h": "down_proj",
+                    "dense_h_to_4h": "gate_up_proj",
+                    "dense": "o_proj",
+                    "query_key_value": "qkv_proj",
+                }
+                for old, new in replacements.items():
+                    split_assignment_str = split_assignment_str.replace(f".{old}", f".{new}")
```

Comment:
Why do you need to do this replacements?

Reply:
The modules are named differently in the model builder. they standardized the names regardless of the source model.

Reply:
Removed this changed and added a note instead